src/serviceFacades/apps.js

```diff
@@ -32,10 +31,7 @@ const ADMIN_APP_DETAILS_QUERY_KEY = "fetchAppDetailsForAdmin";
 const ADMIN_APP_AVU_QUERY_KEY = "fetchAppAVUs";
 
 const getAppTypeFilter = (appTypeFilter) => {
-    const typeFilter =
-        appTypeFilter && appTypeFilter !== appType.all
```

Comment:
This is just me checking -- I guess this change to checking only if `appTypeFilter` is truthy works because `appType.all` isn't actually defined in `AppType.js`, so it's undefined or similar anyway?

(just want to make sure since it otherwise appears this is changing how this code works with that particular value)

Reply:
Yep! I discovered `appType.all` was removed in #143, so I cleaned this up since it's been `undefined` since that change.